core/src/main/scala/org/apache/spark/ui/exec/ExecutorsPage.scala

```diff
@@ -33,11 +33,13 @@ private[ui] case class ExecutorSummaryInfo(
     rddBlocks: Int,
     memoryUsed: Long,
     diskUsed: Long,
+    totalCores: Int,
```

Comment:
Did some checks, MiMa exclude is still needed even without this change and this change did not affect any code, therefore I've removed the change.